### ISSUE TITLE
A few settings commonly found in other terminal emulators

### ISSUE THREAD

**Issue by Karl-D-Asmussen** [lxqt/qterminal#531]
QTerminal looks good, but lacks a couple of otherwise ubiquitous settings.

Almost every other TE available allows the user to:
- Completely disable all scrollback (popular with purists/masochists)
- Specify initial size in terms of character cells

(Using 0.8.0, from Ubuntu upstream)

**Comment by yan12125** [comment]
As a side note, the second feature is discussed in #492 too.

**Comment by cxw42** [comment]
I just recently started using lxqt and am enjoying it!  I would like to request the feature that XTerm has to switch between the regular screen and the alternate screen ([ref](https://invisible-island.net/xterm/xterm.faq.html#xterm_tite)).  I have looked in the manual, issues, Google results, and preferences, and have not been able to find this function.  Thank you for considering this request!

**Comment by yan12125** [comment]
@cxw42 Alternate screens are supported and enabled unconditionally in qtermwidget/qterminal. Mind to share more details for your use cases?

**Comment by cxw42** [comment]
@yan12125 Thanks for asking!  Yes, the alternate screen works fine.  In xterm, I have appreciated the ability to switch back and forth without disrupting application state.  E.g.,

- Normal screen: shell prompt, and the output from a previous `git log` command
- Alternate screen: `less`, showing results of an `ag` search over a large codebase.

By switching between normal and alternate screens, I can refer to the log results in deciding how to use my search results.

Alternatives:

- Ctl-z the `less`, then `fg` back.  However, this scrolls the normal screen.  Therefore, if I switch back and forth several times, information I need on the normal screen may scroll off the top of the window.
- Multiple terminals or tabs.  However, the former requires more screen space, and both require keeping two shell sessions in sync (e.g., same working directory, sometimes same custom env cars).

Please let me know whether or not this example is helpful.  If I am not explaining clearly enough, I can try a different way of conveying the information.  Thanks!

**Comment by yan12125** [comment]
Hmm, the xterm page you've mentioned talks about enable/disable the alternate screen. I guess what you want is somewhat different - switching between normal and alternate screens without Ctrl-Z/fg hacks?

**Comment by cxw42** [comment]
Correct- sorry for the confusion.  I should have quoted from that link:

> xterm allows you ... to switch the display back to the alternate screen to select text from it, to paste into the normal screen

And it works the other way as well (switch to normal then paste into alternate).

**Comment by yan12125** [comment]
Got it! Thanks for the confirmation.

**Comment by luebking** [comment]
https://github.com/lxqt/qterminal/issues/1374

Switching between screen buffers is btw. done via CSI sequences
* alternate: "\033[?1049h\033[H"
* regular: "\033[?25h\033[?1049l"
